test/e2e/subroutes_test.go

```diff
@@ -85,9 +85,11 @@ func TestSubrouteLocalSTS(t *testing.T) { // We can't use a longer more descript
 	// helloworld app and its route are ready. Running the test cases now.
 	for _, tc := range testCases {
 		domain := fmt.Sprintf("%s-%s", tag, resources.Route.Status.Address.URL.Host)
-		helloworldDomain := strings.TrimSuffix(domain, tc.suffix)
+		// TODO(bancel): WTF is this trimming supposed to do?
```

Comment:
It looks like the intent is to test that shorter DNS names work properly? https://github.com/knative/serving/blob/495b381b5fad72bb2d0a3a7b239354fbd5ff5a90/test/e2e/service_to_service_test.go#L59

cc @tcnghia

Reply:
Ok. The test passes, so my change looks correct.